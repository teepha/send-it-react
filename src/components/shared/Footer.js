import React from "react";
import { Link } from "react-router-dom";
import "../../css/styles.css";

const Footer = () => (
  <div>
    <footer>
      <div className="wrapper">
        <ul>
          <li>
            <h4>About US</h4>
          </li>
          <li>SendIT is a courier service.</li>
          <li>Where users can send parcels to different destinations.</li>
        </ul>
        <ul>
          <li>
            <h4>Get in Touch</h4>
          </li>
          <li>Lagos Address</li>
          <li>445 Ikorodu Road</li>
          <li>Illupeju, Lagos</li>
          <li>0814.4523.455</li>
        </ul>
        <ul>
          <li>
            <h4>Working Hours</h4>
          </li>
          <li>Weekdays: 7am to 9pm</li>
          <li>Weekends: 9am to 7pm</li>
          <li>Public Holidays: 10am to 7pm</li>
          <li>Festive Period: 9am to 6pm</li>
        </ul>
        <ul>
          <li>
            <h4>More Info</h4>
          </li>
          <li>
            <Link to="#">Careers</Link>
          </li>
          <li>
            <Link to="#"> Help Center</Link>
          </li>
          <li>
            <Link to="#">Terms&Conditions</Link>
          </li>
          <li>
            <Link to="#">Privacy Policy</Link>
          </li>
        </ul>
        <div id="copyright">Copyright &copy; 2018 SendIT. All rights reserved.</div>
      </div>
    </footer>
  </div>
);

export default Footer;
