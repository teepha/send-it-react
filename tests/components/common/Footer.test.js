import React from "react";
import { shallow } from "enzyme";
import Footer from "../../../src/components/common/Footer";

describe("Footer Component test", () => {
  it("render footer without crashing", () => {
    expect(shallow(<Footer />)).toMatchSnapshot();
  });
});
