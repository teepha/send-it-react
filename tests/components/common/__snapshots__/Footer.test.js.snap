// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component test render footer without crashing 1`] = `
<div>
  <footer>
    <div
      className="wrapper"
    >
      <ul>
        <li>
          <h4>
            About US
          </h4>
        </li>
        <li>
          SendIT is a courier service.
        </li>
        <li>
          Where users can send parcels to different destinations.
        </li>
      </ul>
      <ul>
        <li>
          <h4>
            Get in Touch
          </h4>
        </li>
        <li>
          Lagos Address
        </li>
        <li>
          445 Ikorodu Road
        </li>
        <li>
          Illupeju, Lagos
        </li>
        <li>
          0814.4523.455
        </li>
      </ul>
      <ul>
        <li>
          <h4>
            Working Hours
          </h4>
        </li>
        <li>
          Weekdays: 7am to 9pm
        </li>
        <li>
          Weekends: 9am to 7pm
        </li>
        <li>
          Public Holidays: 10am to 7pm
        </li>
        <li>
          Festive Period: 9am to 6pm
        </li>
      </ul>
      <ul>
        <li>
          <h4>
            More Info
          </h4>
        </li>
        <li>
          <Link
            replace={false}
            to="#"
          >
            Careers
          </Link>
        </li>
        <li>
          <Link
            replace={false}
            to="#"
          >
             Help Center
          </Link>
        </li>
        <li>
          <Link
            replace={false}
            to="#"
          >
            Terms&Conditions
          </Link>
        </li>
        <li>
          <Link
            replace={false}
            to="#"
          >
            Privacy Policy
          </Link>
        </li>
      </ul>
      <div
        id="copyright"
      >
        Copyright Â© 2018 SendIT. All rights reserved.
      </div>
    </div>
  </footer>
</div>
`;
